<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>394</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>394</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MAJENI - Kart Simulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="distance_label">
            <property name="text">
             <string>Distance (km) :</string>
            </property>
            <property name="buddy">
             <cstring>distance</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="distance">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="value">
             <double>1000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="point_label">
            <property name="minimumSize">
             <size>
              <width>72</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>72</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Point (km) :</string>
            </property>
            <property name="buddy">
             <cstring>point</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="point">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="add_point">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>Ajouter</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>170</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>170</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string>Pente</string>
        </property>
        <widget class="QSlider" name="gradient_slide">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>75</y>
           <width>150</width>
           <height>20</height>
          </rect>
         </property>
         <property name="minimum">
          <number>-1000000</number>
         </property>
         <property name="maximum">
          <number>1000000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
         <property name="tickInterval">
          <number>10</number>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="gradient_spin_box">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>27</y>
           <width>81</width>
           <height>22</height>
          </rect>
         </property>
         <property name="correctionMode">
          <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
         </property>
         <property name="minimum">
          <double>-10000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
        <widget class="QRadioButton" name="gradient_degrees">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>30</y>
           <width>41</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>°</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="gradient_radian">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>50</y>
           <width>41</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>rad</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="gradient_pourcent">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>10</y>
           <width>41</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>%</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>75</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>75</height>
           </size>
          </property>
          <property name="title">
           <string>Données</string>
          </property>
          <widget class="QLabel" name="points_label">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>15</y>
             <width>41</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Points :</string>
           </property>
          </widget>
          <widget class="QLabel" name="points">
           <property name="geometry">
            <rect>
             <x>55</x>
             <y>15</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QLabel" name="gradient_min_label">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>35</y>
             <width>61</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Pente min. :</string>
           </property>
          </widget>
          <widget class="QLabel" name="gradient_max_label">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>55</y>
             <width>61</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Pente max. :</string>
           </property>
          </widget>
          <widget class="QLabel" name="gradient_min">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>35</y>
             <width>41</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QLabel" name="gradient_max">
           <property name="geometry">
            <rect>
             <x>85</x>
             <y>55</y>
             <width>41</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="weight">
            <property name="minimumSize">
             <size>
              <width>55</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>55</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>Poids (kg) :</string>
            </property>
            <property name="buddy">
             <cstring>weight_value</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="weight_value">
            <property name="minimumSize">
             <size>
              <width>77</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>77</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximum">
             <double>150000.000000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="simulation">
          <property name="text">
           <string>Simulation temps réel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="erase_point">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Supprimer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTableWidget" name="measures">
      <property name="minimumSize">
       <size>
        <width>650</width>
        <height>150</height>
       </size>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
      <property name="columnCount">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="progressLabel">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="show_bdd_button">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Afficher la BDD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="start_measures">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Démarrer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stop_measures">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Arrêter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="results_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Résultats</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="minimumSize">
       <size>
        <width>650</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>23</height>
       </size>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <widget class="QMenu" name="new_data">
     <property name="title">
      <string>Nouveau</string>
     </property>
     <addaction name="new_custom_data"/>
    </widget>
    <addaction name="new_data"/>
    <addaction name="open"/>
    <addaction name="separator"/>
    <addaction name="save"/>
    <addaction name="save_as"/>
    <addaction name="save_bdd"/>
    <addaction name="separator"/>
    <addaction name="quit"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="save_as">
   <property name="text">
    <string>Enregistrer sous...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="save_bdd">
   <property name="text">
    <string>Afficher la Base de Données</string>
   </property>
  </action>
  <action name="quit">
   <property name="text">
    <string>Quitter...</string>
   </property>
  </action>
  <action name="new_custom_data">
   <property name="text">
    <string>Configuration personnalisé</string>
   </property>
  </action>
  <action name="save">
   <property name="text">
    <string>Enregistrer...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="open">
   <property name="text">
    <string>Ouvrir...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
